roles/ceph-docker-common/defaults/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
-
 ceph_docker_registry: docker.io
 ceph_docker_enable_centos_extra_repo: false
+
+# Set uid/gid to default '64045' for bootstrap directories.
```

Comment:
+ This value as to be set according to the base OS used by the container image, **not** the host.